script/fs-bench/tools/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/containerd/stargz-snapshotter/script/fs-bench/tools
+
+go 1.13
+
+// DO NOT use `go mod tidy` here, it will probably introduce a problem like:
+// https://github.com/prometheus/prometheus/issues/5590
```

Comment:
Do we use these go tools only inside containers defined in `script/fs-bench/work`? If so, we should move `script/fs-bench/tools` directory under the `script/fs-bench/work` and build these binaries during `docker build` of the container.

Reply:
will do